assets/src/dashboard/animations/parts/zoom/index.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { ANIMATION_TYPES } from '../../constants';
+import SimpleAnimation from '../simpleAnimation';
+
+const defaults = {
+  fill: 'forwards',
+  duration: 1000,
+};
+
+export function AnimationZoom({ zoomFrom = 0, zoomTo = 1, ...args }) {
+  const timings = {
+    ...defaults,
+    ...args,
+  };
+
+  const animationName = `from-${zoomFrom}-to-${zoomTo}-${ANIMATION_TYPES.ZOOM}`;
```

Comment:
In a future PR I would love to see some of these string generators become external and then have a unit test. I do this with routes as just a spot check because strings can easily break.

Reply:
Yes, I like that idea and I need to add in more tests for lots of my added stuff.  I will tackle later this week.